packages/build-utils/test/unit.get-env-for-package-manager.test.ts

```diff
@@ -472,23 +496,26 @@ describe('Test `getPathOverrideForPackageManager()`', () => {
       getPathOverrideForPackageManager({
         cliType: args.cliType,
         lockfileVersion: args.lockfileVersion,
+        // naive assumption that enabling corepack as a feature means it's used, but this is fine for tests
+        corepackEnabled: Boolean(args.env.ENABLE_EXPERIMENTAL_COREPACK),
         nodeVersion: args.nodeVersion,
-        env: args.env,
       })
     ).toStrictEqual(want);
   });
 });
+
 describe('Test `getPathForPackageManager()`', () => {
   test.each<{
     name: string;
-    args: Parameters<typeof getEnvForPackageManager>[0];
+    args: Parameters<typeof getPathForPackageManager>[0];
```

Comment:
These types happened to be the same, but this code was referencing the wrong function.

Reply:
good catch!